Added questions, feedback and contributions section to README (#383)
SonarSource/docker-sonarqube: README.md

[![Build Status](https://travis-ci.org/SonarSource/docker-sonarqube.svg)](https://travis-ci.org/SonarSource/docker-sonarqube)

Have Question or Feedback?
--------------------------

For support questions ("How do I?", "I got this error, why?", ...), please first read the [documentation](https://docs.sonarqube.org) and then head to the [SonarSource forum](https://community.sonarsource.com/). There are chances that a question similar to yours has already been answered. 

Be aware that this forum is a community, so the standard pleasantries ("Hi", "Thanks", ...) are expected. And if you don't get an answer to your thread, you should sit on your hands for at least three days before bumping it. Operators are not standing by. :-)

Contributing
------------

If you would like to see a new feature, please create a new thread in the forum ["Suggest new features"](https://community.sonarsource.com/c/suggestions/features).

> Please be aware that we are not actively looking for feature contributions. The truth is that it's extremely difficult for someone outside SonarSource to comply with our roadmap and expectations. Therefore, we typically only accept minor cosmetic changes and typo fixes.

Reviewer: It's a bit reductive to say that only typos and cosmetic contributions are accepted. I suggest to replace by `Therefore, we typically only accept minor improvements and bug-fixes`.

Reviewer 2: I would even suggest we simplify the whole paragraph:
```suggestion
Please be aware that we are not actively looking for feature contributions. We typically accept minor improvements and bug-fixes.
```

With that in mind, if you would like to submit a code contribution, please create a pull request for this repository. Please explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make.

> Make sure that you follow our [code style](https://github.com/SonarSource/sonar-developer-toolset#code-style) and all tests are passing.

Reviewer: I'm not sure that it really applies to this Docker repository. I suggest to drop this paragraph.

Reviewer 2: +1

### License

> Copyright 2015-2019 SonarSource.

Reviewer: That's the opportunity to replace with 2015-2020.
